src/operator/tensor/broadcast_reduce_op.h

```diff
@@ -566,7 +566,7 @@ struct SumCsrKernel<req, 1> {
   }
 };
 
-template <typename xpu>
+template <typename xpu, bool normalize = false>
```

Comment:
Please add brief comment regarding what normalize is for

Reply:
Added comment!